src/Farmhand.js

```diff
@@ -403,6 +408,7 @@ export default class Farmhand extends Component {
         [toolType.WATERING_CAN]: toolLevel.DEFAULT,
       },
       useAlternateEndDayButtonPosition: false,
+      hideHomeScreen: false,
```

Comment:
I feel that the semantics for the Settings view should indicate positive/affirmative states ("on" instead of "off", "yes" instead of "no"), as it creates a more consistent user experience. Considering that, would you mind changing `hideHomeScreen: false` to `showHomeScreen: true` and updating the relevant logic in this PR to match?

Additionally, please move this to property to its alphabetical position in the object.

Reply:
Will do and I'll make a note so I don't forget if there are more settings added later. :) Fixed and alphabetized!